test/integration/full/landmark-no-more-than-one-main/frames/level1.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html id="pass2">
+<head>
+	<meta charset="utf8">
+	<script src="/axe.js"></script>
+</head>
+<body>
+	<p>No main content should still pass</p>
```

Comment:
Hi @jongund, I think the concept is fine, although I think, given that you're also proposing an "at least one main" rule, those two can be part of the same rule. Have these two checks, but make them part of the same rule. What do you think?